tests/conftest.py

```diff
@@ -16,15 +19,22 @@ def client():
 def sessions():
     Base.metadata.create_all(bind=engine)
     sessions = SessionLocal()
-    User1 = User(username="Pure", email="ttxdxp@gmail.com")
+    yield sessions
+    Base.metadata.drop_all(bind=engine)
+
+
+@pytest.fixture
+@given(email=emails())
+def user(sessions, email):
```

Comment:
Same comment as in [Idan's PR](https://github.com/PythonFreeCourse/calendar/pull/44) (BTW, enjoy solving conflicts :) ):
Consider: create a separate db_entities conftest file.

As far as I know, this is how you reference the other files:
pytest_plugins = ['module1', 'module2']
([stackoverflow answer](https://stackoverflow.com/questions/27064004/splitting-a-conftest-py-file-into-several-smaller-conftest-like-parts))

Reply:
I don't see any problem with how I referenced the fixtures.
Could you elaborate on why it seems not good enough for you?
Sessions is something that anyone could use, it is just for referencing the database.